src/Components/common/LoadingState/Loading/index.js

```diff
@@ -0,0 +1,10 @@
+import { IconSpinner, LoadingText, IconContainer } from "./styled";
+
+export const Loading = () => (
+  <>
+    <LoadingText>Search results for “Mulan”</LoadingText>
+    <IconContainer>
+      <IconSpinner />
+    </IconContainer>
```

Comment:
Myślę, że te nazwy od komponentu lub elementu są  tu też nie potrzebne.

Reply:
the same here